lib/modules/manager/npm/utils.ts

```diff
@@ -0,0 +1,20 @@
+import detectIndent from 'detect-indent';
+import { logger } from '../../../logger';
+import type { LockFile, ParseLockFileResult } from './types';
+
+export function parseLockFile(lockFile: string): ParseLockFileResult {
+  const detectedIndent = detectIndent(lockFile).indent || '  ';
+
+  let lockFileParsed: any;
```

Comment:
```suggestion
  let lockFileParsed: LockFile;
```

Reply:
had to adapt this and allow undefined, because it's used without being assigned. let me know if you have better ideas, I'm happy adapting it